.travis.yml

```
@@ -2,4 +2,3 @@ language: node_js
 node_js:
```

may as well add 14 in I suppose
```suggestion
node_js:
  - "14"
```